### ISSUE TITLE
License File Missing?

### ISSUE THREAD

**Issue by dotMR** [GrapeCity/DsExcel-Java#2]
Hello GrapeCity team,

There is no license file in this repository - is it intended to be open source for external usage?

**Comment by shangpeixu** [comment]
Hi Mario,
Thank you for paying attention to  GcExcel Java, this product is not open source library, we just put some demos in this repository.

**Comment by syifarahmat** [comment]
apakah saya bisa menggunakan GcExxel Java dengan lisensi yg saya peroleh dari spreadJs,

''
can I use GcExxel Java with the license that I got from spreadJs

**Comment by shangpeixu** [comment]
No, you need to purchase GcExcel separately, you can find more details about GcExcel at https://www.grapecity.com/documents-api-excel-java
